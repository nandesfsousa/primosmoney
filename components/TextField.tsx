import * as React from 'react';

import { Input, InputProps } from './Themed';

export function TextField(props: InputProps) {
  return <Input {...props}/>;
}
