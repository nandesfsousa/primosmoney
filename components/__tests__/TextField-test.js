import * as React from 'react';
import renderer from 'react-test-renderer';

import { TextField } from '../TextField';

it(`renders correctly`, () => {
  const tree = renderer.create(<TextField value="" onChange={()=>{}}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
