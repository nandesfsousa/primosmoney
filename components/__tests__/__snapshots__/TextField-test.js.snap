// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  onChange={[Function]}
  placeholderTextColor="#ffe002"
  rejectResponderTermination={true}
  style={
    Array [
      Object {
        "color": "#ffe002",
      },
      undefined,
    ]
  }
  underlineColorAndroid="#ffe002"
  value=""
/>
`;
